core/atomic/int64.go

```diff
@@ -0,0 +1,18 @@
+package atomic
```

Comment:
why not using directly atomic? The rest of the project uses atomic directly.

Reply:
Actually, the abstractions in `core/atomic` make the code a bit more readable and less error-prone regarding atomic operations (for example, higher level semantics such as `flag` and `counter` are always better than raw int variables). We had used the abstractions when implementing `TxCache` counters and so on. The rest of the project should use these abstraction to improve code readability.

Reply:
Ok, then we need those strucs tested.